controllers/nstemplateset/nstemplateset_controller_test.go

```diff
@@ -1722,89 +1734,156 @@ func withTemplateRefUsingRevision(revision string) objectMetaOption {
+				"abcde12": test.CreateTemplate(test.WithObjects(ns, rb, role, rbacRb), test.WithParams(username)),
+			},
+			"stage": {
+				"abcde11": test.CreateTemplate(test.WithObjects(ns, rb, role, rbacRb), test.WithParams(username)),
+				"abcde12": test.CreateTemplate(test.WithObjects(ns, rb, role, rbacRb), test.WithParams(username)),
+			},
+			"admin": { // space roles
+				"abcde11": test.CreateTemplate(test.WithObjects(spaceAdmin, spaceAdminRb), test.WithParams(namespace, username)),
+			},
+		},
+		"withemptycrq": {
+			"clusterresources": {
+				"abcde11": test.CreateTemplate(test.WithObjects(advancedCrq, emptyCrq, clusterTektonRb), test.WithParams(username)),
+				"abcde12": test.CreateTemplate(test.WithObjects(advancedCrq, emptyCrq, clusterTektonRb), test.WithParams(username)),
+			},
+			"code": {
+				"abcde11": test.CreateTemplate(test.WithObjects(ns, rb, role), test.WithParams(username)),
+				"abcde12": test.CreateTemplate(test.WithObjects(ns, rb, role), test.WithParams(username)),
+			},
+			"dev": {
+				"abcde11": test.CreateTemplate(test.WithObjects(ns, rb, role), test.WithParams(username)),
+				"abcde12": test.CreateTemplate(test.WithObjects(ns, rb, role), test.WithParams(username)),
+			},
+			"stage": {
+				"abcde11": test.CreateTemplate(test.WithObjects(ns, rb, role), test.WithParams(username)),
+				"abcde12": test.CreateTemplate(test.WithObjects(ns, rb, role), test.WithParams(username)),
+			},
+			"admin": { // space roles
+				"abcde11": test.CreateTemplate(test.WithObjects(spaceAdmin, spaceAdminRb), test.WithParams(namespace, username)),
+			},
+		},
+		"appstudio": {
+			"clusterresources": {
+				"abcde11": test.CreateTemplate(test.WithObjects(advancedCrq, clusterTektonRb, idlerDev, idlerCode, idlerStage), test.WithParams(username)),
+			},
+			"appstudio": {
+				"abcde11": test.CreateTemplate(test.WithObjects(ns, rb, role, rbacRb), test.WithParams(username)),
+			},
+			"admin": { // space roles
+				"abcde11": test.CreateTemplate(test.WithObjects(spaceAdmin, spaceAdminRb), test.WithParams(namespace, username)),
+			},
+			"viewer": { // space roles
+				"abcde11": test.CreateTemplate(test.WithObjects(spaceViewer, spaceViewerRb), test.WithParams(namespace, username)),
+			},
+		},
+	}
+	for tierName, tierTmpls := range tmpls {
+		for typeName, typeTmpls := range tierTmpls {
+			for revision, tmpl := range typeTmpls {
+				tierTmpl, err := createTierTemplate(decoder, tmpl, tierName, typeName, revision)
 				if err != nil {
 					return nil, err
 				}
-				tierTmpl := &toolchainv1alpha1.TierTemplate{
-					ObjectMeta: metav1.ObjectMeta{
-						Name:      strings.ToLower(fmt.Sprintf("%s-%s-%s", tierName, typeName, revision)),
-						Namespace: test.HostOperatorNs,
-					},
-					Spec: toolchainv1alpha1.TierTemplateSpec{
-						TierName: tierName,
-						Type:     typeName,
-						Revision: revision,
-						Template: tmpl,
-					},
-				}
+				t.Logf("adding tiertemplate '%s", tierTmpl.Name)
```

Comment:
this produces quite a lot of lines in the test log and (I guess that) they are not super useful 
```suggestion
```

Reply:
sure, but those logs will only be printed if the test fails, AFAIK